app/os_js.go

```diff
@@ -2,6 +2,7 @@
 
 package app
 
+import "C"
```

Comment:
What's this for? It doesn't seem to be used anywhere.

Reply:
Fixed